src/Util/Network.php

```diff
@@ -36,7 +36,13 @@ class Network
 	 */
 	public static function fetchUrl($url, $binary = false, &$redirects = 0, $timeout = 0, $accept_content = null, $cookiejar = 0)
 	{
-		$ret = self::curl(
+		$ret = fetchUrlFull($url, $binary, $redirects, $timeout, $accept_content, $cookiejar);
+
+		return($ret['body']);
+	}
+	public static function fetchUrlFull($url, $binary = false, &$redirects = 0, $timeout = 0, $accept_content = null, $cookiejar = 0)
```

Comment:
Please add a space before the function declaration, as well as a Doxygen doc block similar to the one above the function declaration you're taking from.